silo/tests/test_views.py

```diff
@@ -1561,3 +1677,68 @@ def test_share_silo_without_owner_failed_for_user(
             messages.append(m.message)
 
         self.assertIn('Invalid Form', messages)
+
+    @patch('silo.views.get_workflowlevel1s')
+    @patch('silo.forms.get_workflowteams')
```

Comment:
Because silo edit page using silo forms and silo forms also calling get_workflowteams function. This is why other silo edit tests including this patch.

Reply:
Ok. I'll check it again. Thanks :)